src/main/java/com/Helpers/Action.java

```diff
@@ -0,0 +1,106 @@
+package com.Helpers;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+import com.base.Base;
+
+public class Action extends Base {
+
+	public void click(WebElement element) {
+
+		WebElement ELEMENT = wait.withMessage("Element is Not Clickable")
+				.until(ExpectedConditions.elementToBeClickable(element));
+		ELEMENT.click();
+	}
+
+	public void type(WebElement element, String Testdata) {
+		try {
+			WebElement ELEMENT = wait.until(ExpectedConditions.visibilityOf(element));
+			ELEMENT.clear();
+			ELEMENT.sendKeys(Testdata);
+		} catch (NullPointerException e) {
```

Comment:
if you use element as variable in the parameter same element should be used you used different ones in the parameters and in functions